apps/api/src/modules/providers/mailgun/mailgun.service.ts

```diff
@@ -80,4 +89,16 @@ export class MailgunService {
       }),
     );
   }
+
+  async getDeliverStatus(messageId: string, providerId: number): Promise<DomainEvent> {
```

Comment:
Maybe a typo, but can you recheck if you meant to type `getDeliveryStatus`?

Can update in the method call as well in consumer.

Reply:
Fixed the typo.